package.json

```diff
@@ -26,8 +26,8 @@
     ]
   },
   "scripts": {
-    "build": "turbo run build --filter=!polaris.shopify.com && turbo run build --filter=polaris.shopify.com",
```

Comment:
Building the site requires the polaris components to be build first. In _most_ cases, `turbo` has been completing the build of the polaris components first, but in _some_ cases, it was running them in parallel (it runs all build tasks in parallel by default), resulting in errors building the site because the polaris components weren't available yet.

The only sure-fire way to order the build correctly was to use this filter syntax, otherwise we'll _occasionally_ run into build errors where the polaris components haven't finished before turbo decides to do the website (eg; a process thread becomes available, etc).

Reply:
I understand, I added a next step as I was concerned about that. We can change the script in `polaris-site-container-builder` so that the build stays simple in this repo and is ran specifically for the build/deploy
https://github.com/Shopify/polaris-site-container-builder/blob/0e1a0e3196b049784706ea5df8ca027025efdbe9/.shopify-build/polaris-site-container-builder-production-builder.yml#L21

Reply:
Interesting. I would like to try the updated versions of turbo. This might be resolved by the version bump. If it does happen again locally I would definitely be happy to add this back.

Reply:
It seems like y'all need to teach turbo repo about the dependencies between projects so that it can execute in a required order - https://turbo.build/repo/docs/core-concepts/monorepos/running-tasks#in-a-different-workspace